backends/qualcomm/quantizer/quantizer.py

```diff
@@ -22,14 +22,17 @@
 from torch.ao.quantization.quantizer import Quantizer
 from torch.fx import GraphModule
 
-from .utils import (
+from .annotators import OP_ANNOTATOR
+
+from .qconfig import (
+    get_16a16w_qnn_ptq_config,
     get_16a4w_qnn_ptq_config,
+    get_16a4w_qnn_qat_config,
     get_16a8w_qnn_ptq_config,
-    get_default_16bit_qnn_ptq_config,
```

Comment:
Removing `get_default_16bit_qnn_ptq_config` causing internal failure...can we do it in a the following way
1. This PR includes both new config and the old config
2. I submit a PR internally to remove the old config call site
3. You submit a new PR to remove the old config

Then we can land it safely...

Reply:
Hi Chen,
Thanks for pointing out what to fix.
It seems to fail again. Shuold I also add back `get_default_8bit_qnn_ptq_config`?

Reply:
Sorry what is fail again?

Reply:
Two  checks in CI chcek list below
<img width="562" alt="image" src="https://github.com/user-attachments/assets/8f2a8a3a-1fe0-4e78-a276-9989ac5796f5">

Reply:
Oh hmm I feel like the `Meta Internal-Only` check isn't very accurate...

Reply:
No problem. Let me know what should be fix if any. :D

Reply:
Let me import and check CI again...